codechain/rpc/base.py

```diff
@@ -0,0 +1,37 @@
+import json
+from typing import Callable
+from typing import Union
+
+import requests
+
+
+class rpc_base:

```

Comment:
If you decide to use `jsonrpcclient`, then this class would be some useless class. It supports automatic id generation, json deserialization, error handlings.